src/features/welcome/welcome-page.js

```diff
@@ -1,17 +1,181 @@
+            memberImageAlt=""
+            memberPosition="Lead Developer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Luisa Rueda"
+            memberImage="https://picsum.photos/200?random=6"
+            memberImageAlt=""
+            memberPosition="Developer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Karen Garcia"
+            memberImage="https://picsum.photos/200?random=7"
+            memberImageAlt=""
+            memberPosition="Developer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Mita Trisnodjojo"
+            memberImage="https://picsum.photos/200?random=8"
+            memberImageAlt=""
+            memberPosition="Lead Designer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Karen Chiu"
+            memberImage="https://picsum.photos/200?random=9"
+            memberImageAlt=""
+            memberPosition="Designer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Dexter Bolasoc"
+            memberImage="https://picsum.photos/200?random=10"
+            memberImageAlt=""
+            memberPosition="Designer"
+            memberLinkedin=""
+          />
+          <TeamMember
+            memberName="Milly Tsou"
+            memberImage="https://picsum.photos/200?random=11"
+            memberImageAlt=""
+            memberPosition="Designer"
+            memberLinkedin=""
+          />
+        </section>
+        <section className="contactUs">
+          <div>
+            <h2 className="titleBlack">Contact us</h2>
+            <p>We're here to help and answer any questions you might have. We look forward to hearing from you.</p>
+          </div>
+          <form action="">
+            <TextField
+              className="firstName"
+              id="outlined-password-input"
+              label="First Name"
+              type="text"
+              placeholder="First Name"
+              margin="normal"
+            />
+            <TextField
+              className="lastName"
+              id="outlined-password-input"
+              label="Last Name"
+              type="text"
+              placeholder="Last Name"
+              margin="normal"
+            />
+            <TextField
+              className="email"
+              id="outlined-password-input"
+              label="Email"
+              type="email"
+              placeholder="Email"
+              fullWidth="true"
+              margin="normal"
+            />
+            <TextField
+              className="message"
+              id="outlined-textarea"
+              label="Message"
+              type="text"
+              placeholder="Message"
+              fullWidth="true"
+              margin="normal"
+              multiline="true"
+              minRows="4"
+            />
```

Comment:
Bolean properties like `fullWidth` and `multiline` should have its value not has a string but as a boolean and when the value is true, it is recommended to use the shorter way as follows:
```suggestion
            <TextField
              className="message"
              id="outlined-textarea"
              label="Message"
              type="text"
              placeholder="Message"
              fullWidth
              margin="normal"
              multiline
              minRows="4"
            />
```